src/character.h

```diff
@@ -2041,10 +2028,10 @@ class Character : public Creature, public visitable<Character>
         units::mass bodyweight() const;
         // returns total weight of installed bionics
         units::mass bionics_weight() const;
-        // increases the activity level to the next level
+        // increases the activity level to the specified level
         // does not decrease activity level
-        void increase_activity_level( float new_level );
-        // decreases the activity level to the previous level
+        void set_activity_level( float new_level );
```

Comment:
I don't think this is quite the right name.  How about `log_activity_level`?

Reply:
log_activity_level() is already taken.